projects.md

```diff
@@ -36,8 +36,14 @@ The following are projects that implement the [OpenPGP](https://en.wikipedia.org
 - [LEAP](https://leap.se)
 - ...to be added...
 
+### APIs and Language Bindings
+
+- [PyME](https://bitbucket.org/malb/pyme) - Python 2 bindings for GPGME
+- [Python GnuPG](https://bitbucket.org/vinay.sajip/python-gnupg)
```

Comment:
actually, i am not sure I approve this anymore: were those projects consulted before agreeing to appear here?

according to the top of the page description, this is not just a mere list of projects using PGP - it's projects explicitely willing to collaborate over the modern pgp project.

i made the same mistake with caff and geysigning - i'm checking with them to see if it's okay they are here.